src/SDL_assert.c

```diff
@@ -245,7 +245,9 @@ static SDL_AssertState SDLCALL SDL_PromptAssertion(const SDL_AssertData *data, v
             state = (SDL_AssertState)selected;
         }
     } else {
-#ifdef SDL_PLATFORM_EMSCRIPTEN
+#ifdef SDL_PLATFORM_PRIVATE
+        #include "SDL_promptassertion_private.h"
```

Comment:
```suggestion
        SDL_PrivatePromptAssertion(...);
```

Reply:
I ended up using a macro for this since it worked better with the build system. Same deal for the gamepad_db header.